packages/p2p/src/components/buy-sell/buy-sell-header.scss

```diff
@@ -79,7 +79,26 @@
             border-radius: 4px;
             display: flex;
             margin-left: 0.8rem;
-            padding: 1.3rem;
+
+            &-icon {
+                align-self: center;
+                display: flex;
+                padding: 1.3rem;
+            }
+
+            &-status {
+                position: absolute;
+                border-radius: 50px;
+                background: #ff444f;
```

Comment:
can we take the color from themes instead of hardcoded value?

Reply:
suree!